Utils to update Artifacts.toml (#134)
beacon-biosignals/Ray.jl: README.md

Ray versions before 2.6 only support pydantic 1.x; downgrade with `pip install "pydantic<2"`.

## Setup

> Ultimately our aim is to make installing Ray.jl as simple as `Pkg.add("Ray")`.  However, for the time being, it's still necessary to install the full build toolchain required to use Ray _and_ our Ray.jl wrappers.

Reviewer: ```suggestion
Ultimately our aim is to make installing Ray.jl as simple as `Pkg.add("Ray")`. However, for the time being, it's still necessary to install the full build toolchain required to use Ray _and_ our Ray.jl wrappers.
```

(These are the same as the [upstream Ray instructions](https://docs.ray.io/en/releases-2.5.1/ray-contribute/development.html#prepare-the-python-environment) but copied here for consistency.)

### Install Ray CLI/server

> We currently rely on a patched version of upstream Ray server/CLI that is aware of Julia as a supported language and knows how to launch julia worker processes.  Until these changes are upstreamed to the Ray project, you need to either [build from source]() or install using our custom-build wheels.

Author: fixed

> We currently rely on a patched version of upstream Ray server/CLI that is aware of Julia as a supported language and knows how to launch julia worker processes.  Until these changes are upstreamed to the Ray project, you need to either [build from source](https://github.com/beacon-biosignals/ray/blob/beacon-main/python/README-building-wheels.md) or install using our custom-build wheels.

Reviewer: ```suggestion
We currently rely on a patched version of upstream Ray server/CLI that is aware of Julia as a supported language and knows how to launch julia worker processes.
Until these changes are upstreamed to the Ray project, you need to either [build from source](https://github.com/beacon-biosignals/ray/blob/beacon-main/python/README-building-wheels.md) or install using our custom-built wheels.
```

NOTE: make sure you've [activated the appropriate virtual environment](#prepare-python-environment) where you want to install the Ray CLI!

#### Install from github release

> Find the appropriate binary wheel for your python version and system (currently supported are ARM MacOS, e.g. M1, and AND64/x86_64 linux) from the [releases page](https://github.com/beacon-biosignals/ray/releases), and install from the release URL.  For instance, to install Ray CLI for Linux running Python 3.9, run this command in the appropriate virtual environment:

Reviewer: ```suggestion
Find the appropriate binary wheel for your python version and system (currently supported are macOS ARM64 (M1), and Linux AMD64/x86_64) from the [releases page](https://github.com/beacon-biosignals/ray/releases), and install from the release URL.
For instance, to install Ray CLI for Linux running Python 3.9, run this command in the appropriate virtual environment:
```

```sh
pip install -U "ray[default] @ https://github.com/beacon-biosignals/ray/releases/download/ray-2.5.1+1/ray-2.5.1-cp39-cp39-manylinux2014_x86_64.whl"
```

Replace the `https://` URL with the approriate release asset URL.
Note: you may also need to increment the build number to get the latest wheels.

(Installing `ray[default]` will also install additional dependencies that are necessary for the cluster status server and dashboard)

#### Build from source

Follow [the instructions](https://github.com/beacon-biosignals/ray/blob/beacon-main/python/README-building-wheels.md) outlined in the Beacon fork of Ray.

### JLL Artifacts

> The `ray_julia_jll` artifacts are hosted in [GitHub releases](https://github.com/beacon-biosignals/Ray.jl/releases) and will be downloaded automatically for any supported platform when calling `Pkg.build()`.

Reviewer: I don't think `Pkg.build` triggers. I may be wrong though
